### ISSUE TITLE
showOnRightClick on Windows opens the taskbar menu + skipTaskbar

### ISSUE THREAD

**Issue by vkolova** [amaurymartiny/menubar#16]
When using `showOnRightClick`, on Windows when you right click the tray icon, the menu that's opened on the taskbar right-click opens on top of the menu window.

![menu](http://prikachi.com/images/798/9570798O.png)

Electron 5 introduced `skipTaskbar` option for BrowserWindow, I'd be nice to be able to use it, since currently the window is shown in the taskbar.

**Comment by amaurymartiny** [comment]
As written in https://github.com/amaurymartiny/menubar/issues/14#issuecomment-493376847, you can either use the hack `new Menubar({ skipTaskbar: true })`, or wait for #17 (I'll get to it this weekend)

**Comment by amaurymartiny** [comment]
In [`@amaurymartiny/menubar@6.1.0`](https://www.npmjs.com/package/@amaurymartiny/menubar), you can do: 

```javascript
const mb = menubar({
  browserWindow: {
    skipTaskbar: true
  }
});
```

Does this solve your issue?

**Comment by vkolova** [comment]
Yes, looks good. I still have the issue with the taskbar menu though.
